Main view has only 3 buttons for reporting

DigitalSeneca opened the pull request:
fixes #100

- I would not merge it until the PR with the drawer is also ready, because the "My Reports" button would be gone otherwise
- I would need to receive the translation string for "What's in your surroundings?" or "What would you like to report?"

![image](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/assets/157690872/f50282b1-bed5-4a50-885c-ce73c4b69689)

DigitalSeneca commented on lib/pages/main/main_vc.dart:
```diff
@@ -180,202 +177,39 @@ class _MainVCState extends State<MainVC> {
+                                  CustomCard(
+                                    text: 'single_mosquito',
+                                    image_path: 'assets/img/ic_mosquito_report.png',
+                                    color: '40DFD458',
```
First two characters indicate the opacity (in this case, 40) and the rest of characters indicate the color

DigitalSeneca commented on lib/pages/main/components/custom_card_widget.dart:
I have removed the code in CustomCard entirely and added new code, since this was only used in HomePage

epou commented:
Great! Could you add some left margin to the images so that it matches the thickness of the top and bottom margins?

DigitalSeneca commented:
> Great! Could you add some left margin to the images so that it matches the thickness of the top and bottom margins?

I've added a left padding of 5.0 and updated the screenshot in the screenshot. Now the image matched the thickness better, but I can still update it as needed

DigitalSeneca commented:
closed in favor of #143 (stacked pull request with target to drawer instead of v34)
